lib/modules/commentDepth.js

```diff
@@ -4,13 +4,10 @@ import { insertParam, regexes } from '../utils';
 export const module = {};
 
 module.moduleID = 'commentDepth';
-module.moduleName = 'Custom Comment Depth';
-module.category = 'Comments';
+module.moduleName = 'commandDepthName';
+module.category = 'commentsCategory';
 module.disabledByDefault = true;
-module.description = `
-	Allows you to set the preferred depth of comments you wish to see when clicking on comments links.
-	0 = Everything, 1 = Root level, 2 = Responses to root level, 3 = Responses to responses to root level, etc.
-`.trim();
+module.description = 'commandDepthDesc'.trim();
```

Comment:
Remove the `trim()`, it was previously there to get rid of the newline at the beginning and end but those are gone now that it's translated